examples/3d/lightmaps.rs

```diff
@@ -36,7 +36,8 @@ fn add_lightmaps_to_meshes(
     let exposure = 250.0;
     for (entity, name, material) in meshes.iter() {
         if &**name == "Light" {
-            materials.get_mut(material).unwrap().emissive = Color::Srgba(Srgba::WHITE * exposure);
+            materials.get_mut(material).unwrap().emissive =
+                LinearRgba::from(Srgba::WHITE) * exposure;
```

Comment:
The exposure multiplier needs to be in Srgba space: move it inside the brackets.